src/index.js

```diff
@@ -51,7 +49,7 @@ const ApolloClientPlugin = createPlugin({
     authKey = 'token',
     includeCredentials = 'same-origin',
     apolloContext,
-    apolloLink,
+    getApolloLinks,
```

Comment:
I am wondering if we can simplify the implementation:
```js
// Default function arg for GetApolloClientLinksToken
getApolloLinks = links => links;

const links = getApolloLinks([authMiddleware, connectionLink]);
```

What do you think?

Reply:
@KevinGrandon I think this is fine ! We pass total control to outside, it's correct approach.